packages/scss/src/components/readMore/component.scss

```diff
@@ -0,0 +1,54 @@
+@use '@lucca-front/scss/src/commons/utils/namespace';
+
+@use '@lucca-front/scss/src/commons/utils/a11y';
+
+@mixin component($atRoot: namespace.$defaultAtRoot) {
+	position: relative;
+	display: block;
+
+	@at-root ($atRoot) {
+		.readMore-link.link {
+			position: absolute;
+			inset: auto 0 var(--pr-t-spacings-50) auto;
+			display: var(--components-readMore-link-display);
+			flex-direction: column;
+			z-index: 1;
+			line-height: var(--sizes-M-lineHeight);
+			background-image: linear-gradient(
+				to right,
+				transparent,
+				var(--components-readMore-link-backgroudColor) calc(var(--components-readMore-link-paddingInlineStart) - var(--pr-t-spacings-100))
+			);
+			padding-inline-start: var(--components-readMore-link-paddingInlineStart);
+			font-weight: 600;
+			text-decoration: none;
+		}
+
+		.readMore-textFlow {
+			display: -webkit-box;
+			-webkit-line-clamp: var(--components-readMore-lineClamp);
+			-webkit-box-orient: vertical;
+			overflow: hidden;
+			padding: var(--pr-t-spacings-50);
+			margin: calc(var(--pr-t-spacings-50) * -1);
```

Comment:
Why do we need to add extra padding / margin? 🤔

Reply:
The component has an overflow and I still wanted the elements inside to have their visible focus fully displayed.

I wanted to avoid this:
![Capture d’écran 2025-03-27 à 10 26 03](https://github.com/user-attachments/assets/0e5a7087-c921-446c-8119-c2835e28cdf5)